src/components/NurseryWithdrawals/PlantingProgressMap.tsx

```diff
@@ -73,15 +86,16 @@ export default function PlantingProgressMap({ plantingSiteId }: PlantingProgress
       if (updateStatus.status === 'error') {
         snackbar.toastError(strings.GENERIC_ERROR);
       } else if (updateStatus.status === 'success') {
-        // refresh planting site data to get new completed state for subzone
+        // refresh planting site and observations data
         dispatch(requestPlantingSites(org.selectedOrganization.id));
+        dispatch(requestObservationsResults(org.selectedOrganization.id));
       }
     }
   }, [updateStatus, dispatch, snackbar, org.selectedOrganization.id]);
 
-  const updatePlantingComplete = useCallback(
-    (id: number, val: boolean) => {
-      // TODO: warn if undoing planting complete will erase statistics
+  const [confirmDeleteCb, setConfirmDeleteCb] = useState<() => void>(() => () => null);
```

Comment:
can we leverage onSubmit to call updatePlantingComplete, without this additional callback?